.github/workflows/release.yaml

```diff
@@ -0,0 +1,135 @@
+# Copyright the Hyperledger Blockchain Explorer contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Build docker image
+
+on:
+  workflow_dispatch: # workflow_dispatch must be enabled in main branch to support release action on older release branches
+  create:
+    tags:
+      - v*
+  push:
+    branches:
+      - main
+    paths:
+      - 'Dockerfile'
+      - 'postgres-Dockerfile'
+
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_PATH: ghcr.io/${{ github.repository_owner }}
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  # on push to main branch, create docker tags latest 
+  build-and-push-docker-latest:
+    name: Build and Push docker image with latest tage
+    if: ${{ startsWith(github.ref, 'refs/heads/main') }}
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Login to the container registry
+      uses: docker/login-action@v1
+      with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Lowercase IMAGE_PATH
+      id: image_path_format
+      uses: ASzc/change-string-case-action@v2
+      with:
+        string: ${{ env.IMAGE_PATH }}
+
+    - name: Build and push img with latest tag
+      id: explorer_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        push: true
+        platforms: linux/amd64,linux/arm64,linux/arm/v7 
+        tags: ${{ steps.image_path_format.outputs.lowercase }}/explorer:latest
+    
+    - name: Build and push img with latest tag
+      id: explorer-db_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./postgres-Dockerfile
+        push: true
+        platforms: linux/amd64,linux/arm64,linux/arm/v7 
```

Comment:
I tried to run release.yaml pipeline and encountered with the below issue.  
![image](https://github.com/hyperledger-labs/blockchain-explorer/assets/65538431/afc9a175-f562-4a6c-9640-bcf7027d9735)

Can you please look into this issue

Reply:
removed the `linux/arm/v7` builds